Gruntfile.js

```diff
@@ -0,0 +1,20 @@
+var globalThreshold = 50;  // Global code coverage threshold (as a percentage)
+
+module.exports = function(grunt) {
+  grunt.initConfig({
+    // Configuration to be run (and then tested).
+    blanket_qunit: {
+      default_options: {
+        options: {
+            urls: ['js_tests/tests.html?coverage=true&gruntReport'],
+            globalThreshold: globalThreshold,
+            threshold: 10,
```

Comment:
They work fine in this file because it will never be run in a browser, but I can remove it for the sake of consistency.